### PR TITLE
Styling the index page with article cards

### PR DESCRIPTION
**Pull request by Joberg95**
#### Description
 Articles as cards
Category bar in center of the page
We had to reinstall tailwind

#### What I have learned working on this feature: 
How to use tailwind classes in a smarter way.
## Index: 
image

### CODE REVIEW

File: coverage/.resultset.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
```
Comment (magnus-thor) [inline coverage/.resultset.json]: Coverage folder should be in gitignore

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/javascript/css/application.css [hunk 2]
```diff
@@ -57,4 +57,5 @@
     *
     * @import "utilities/background-patterns";
     * @import "utilities/skew-transforms";
-    */
\ No newline at end of file
+    */
+
```
Comment (aleciabenjamin) [inline app/javascript/css/application.css]: extra line

File: config/environments/development.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.configure do
+  # Verifies that versions and hashed value of the package contents in the project's package.json
```
Comment (NiciP) [inline config/environments/development.rb:2]: Need to remove comments here?

File: config/environments/development.rb [hunk 4]
```diff
@@ -1,4 +1,6 @@
...
+  config.webpacker.check_yarn_integrity = true
   # Settings specified here will take precedence over those in config/application.rb.
```
Comment (NiciP) [inline config/environments/development.rb:4]: Need to remove comments here?

**Review by NiciP — APPROVED** [review]

File: app/javascript/css/application.css (hunk 2 above)
Comment (magnus-thor) [reply app/javascript/css/application.css]: Remove this file

File: app/javascript/css/tailwind.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    font-family: 'Over the Rainbow', cursive;
+    @apply text-5xl; 
+    @apply text-center;
+    @apply mt-8;
+    @apply leading-loose;
+    
+}
+
+.back {
+    margin-left: 25px;
```
Comment (magnus-thor) [inline app/javascript/css/tailwind.css]: this can be change to `@apply ml-6;`

File: app/views/articles/index.html.erb [hunk 6]
```diff
@@ -1,16 +1,33 @@
...
+      </li>
     <% end %>
-    </p>
-<% end %>
\ No newline at end of file
+  </ul>
+</div>
+<div class="article-grid">
+  <% @articles.each do |article|%>
+    <div id="#{dom_id(article)}">
```
Comment (magnus-thor) [inline app/views/articles/index.html.erb]: Change this to `<div id="<%= dom_id(article) %>">`

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]

**Pull request merged**